src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2094,4 +2095,12 @@ Reset\ All=Reset All
 Column\ type\ %0\ is\ unknown.=Column type %0 is unknown.
 Linked\ identifiers=Linked identifiers
 Special\ field\ type\ %0\ is\ unknown.\ Using\ normal\ column\ type.=Special field type %0 is unknown. Using normal column type.
+JabRef\ Parser = JabRef Parser
```

Comment:
Please no space before `=` in JabRef_en.properties

Please inspect the old code if some translation strings could be reused. https://github.com/JabRef/jabref/pull/4717

"JabRef Parser" is a strange dialog heading. Maybe "Plain text import" (https://github.com/JabRef/jabref/pull/4717/files#diff-63c594fa122cbaa34c72e860d42a75efL697)